_drafts/2015-12-16-improving-scrolling-performance-of-a-UICollectionView.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout:       post
+title:        Additional tips on improving scrolling performance of a UICollectionView
+author:       amit_rao
+summary:      Recently we improved the scrolling performance of the WeWork iOS app with the goal of getting as close to 60 fps consistently as possible. The blog post describes my experience solving the issue and adding to some of the existing online resources related to the issue.
+image:		  http://res.cloudinary.com/wework/image/upload/v1430060208/engineering/recruiting-through-gamification.jpg      
+categories:   engineering
+---
+
+#### What does optimizing scrolling performance mean and why is it important? 
```

Comment:
Not sure if you need a header here. the paragraph below is very short and can probably stand on its own as an intro summary of the article. You can even repeat some of what you have in the `summary` section above